src/app/trips/page.tsx

```diff
@@ -0,0 +1,25 @@
+import Header from "@/components/Header";
+import { HeaderVariant } from "@/enums/Header";
+import React from "react";
+
+const header = {
+  title: "Book trip by ExperienceRW",
+  description:
+    "Lorem Ipsum Lorem Ipsum  Lorem Ipsum Lorem Ipsum  Lorem Ipsum Lorem Ipsum  Lorem Ipsum Lorem Ipsum",
+};
```

Comment:
Can you import these data from another file like data file

Reply:
yes